### DISCUSSION TITLE
[Feature] <show metadata without GPS info>

### DISCUSSION THREAD

**Discussion by hellodk34** [immich-app/immich#10581] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello!

Dear immich team, thanks for your hard working, immich works so fine for my daily life, I really like its every feature.

The day before yesterday I created a shared album, and generated a public URL, and put it on my personal website. I want to make this public album an album where I can show my photography works, but many photos are taken by mobile phones, which have GPS information. I want to hide the GPS information but keep the EXIF ​​information.

Now your web application has the function of `show metadata`, but I hope to add a function of `show metadata without GPS info`, so that the album I shared can be used as a photography album. Not only does it remove sensitive address location information, but it can also disclose information such as ISO, aperture, shutter time, lens specifications, and shooting equipment of the photo.

Thank you for your attention and look forward to your response. ^_^

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#9835

**Reply by hellodk34** [reply]
> #9835

ok, thanks.

Sorry for not finding that discussion.
